google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/SpannerMetadataProvider.java

```diff
@@ -66,6 +66,12 @@ Map<String, List<String>> newExtraHeaders(
         .build();
   }
 
+  Map<String, List<String>> newRouteToLeaderHeader() {
+    return ImmutableMap.<String, List<String>>builder()
+        .put(routeToLeaderHeaderKey, Collections.singletonList("true"))
+        .build();
+  }
```

Comment:
You don't need to create a method for this. Instead, you can just create a `static final` field for this, as the contents of the map is always the same (and it is immutable). That saves the creation of a new map every time the option is used.

Reply:
Done